docs/pages/versions/unversioned/sdk/webbrowser.md

```diff
@@ -60,30 +65,82 @@ Opens the url with Safari in a modal on iOS using `SFAuthenticationSession`. The
+
+#### Arguments
+
+- **browserPackage (_string_)** -- **optional** -- package of browser to be cooled. If not set, preferred browser will be used.
+
+#### Returns
+
+The promise resolves with `{ type: 'cooling', package: string }` when cooling is performed, or `{ result: 'Nothing to cool down' }` when there was no connection to be dismissed.
 
 ### `WebBrowser.dismissBrowser()`
 
-Dismisses the system's presented web browser. On Android calling this method doesn't actually do anything, the user still has to manually dismiss the browser.
+_iOS only_
+
+Dismisses the presented web browser.
 
 #### Returns
 
 The promise resolves with `{ type: 'dismiss' }`.
 
 ### `WebBrowser.getCustomTabsSupportingBrowsers`
 
-Available only on Android.
+_Android only_
 
-Returns two lists of applications package names supporting Custom Tabs. Under `packages` key you can find a list of all the applications. Under `default` key only the default applications will be listed. Note that list might not be fully reliable, as it uses `PackageManager.getResolvingActivities`.
+Returns a list of applications package names supporting Custom Tabs, Custom Tabs service, user chosen and prefered one. This may not be fully reliable, since it uses `PackageManager.getResolvingActivities` under the hood. (For example, some browsers might not be present in `browserPackages` list once another browser is set to defult.)
 
 #### Returns
 
-The promise resolves with `{ packages: string[], default: string[] }`
+The promise resolves with `{ browserPackages: string[], defaultPackage: string, servicePackages: string[], preferredPackage: string }`
+
+- **browserPackages (_string[]_)** : All packages recognized by PackageManager as capable of handling Custom Tabs. Empty array means there is no supporting browsers on device.
+- **defaultPackage (_string_)** : Default package chosen by user. Null if there is no such packages. Null usually means, that user will be prompted to choose from available packages.
+- **servicePackages (_string[]_)** : All packages recognized by PackageManager as capable of handling Custom Tabs Service. This service is used by [`warmUpAsync`](#webbrowserwarmupasyncnbrowserpackage), [`mayInitWithUrlAsync`](#webbrowsermayinitwithurlasyncurl-package) and [`coolDownAsync`](#webbrowsercooldownasyncbrowserpackage).
+- **preferredPackage (_string_)** : Package preferred by CustomTabsClient to be used to handle Custom Tabs. It favors browser chosen by user as default, as long as it is present on both `browserPackages` and `servicePackages` lists. Only such browsers are considered as fully supporting Custom Tabs. It might be `null` when there is no such browser installed or when default browser is not on `servicePackages` list.
```

Comment:
```suggestion
- **preferredPackage (_string_)** : Package preferred by `CustomTabsClient` to be used to handle Custom Tabs. It favors browser chosen by user as default, as long as it is present on both `browserPackages` and `servicePackages` lists. Only such browsers are considered as fully supporting Custom Tabs. It might be `null` when there is no such browser installed or when default browser is not in `servicePackages` list.
```